runtime/src/accounts_db.rs

```diff
@@ -448,6 +454,9 @@ pub struct AccountsDB {
     /// Starting file size of appendvecs
     file_size: u64,
 
+    /// Accounts that will cause a panic! if modified
+    frozen_accounts: Mutex<HashMap<Pubkey, FrozenAccountInfo>>,
```

Comment:
I don't love this `Mutex`.   But to get rid if it, I'll need to plumb `frozen_accounts` through `AccountsDb::new()` and then I'm changing code all over the place.